app/assets/javascripts/file_manager.js

```diff
@@ -0,0 +1,137 @@
+function rename(path) {
+    let new_name = prompt("Enter the new name:");
+    let rel_path = path.split("/file_manager/")[1].replace(/%40/g, '@');
```

Comment:
Consider `decodeURIComponent` instead of replacing specific escape codes?

Reply:
Changed to decodeURIComponent